tests/vmi_configuration_test.go

```diff
@@ -2582,23 +2581,21 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				emulator = filepath.Base(emulator)
 
 				virtClient := kubevirt.Client()
-				pscmd := []string{"ps", "-C", emulator, "-o", "pid", "--noheader"}
-				_, err = exec.ExecuteCommandOnPod(
-					virtClient, readyPod, "compute", pscmd)
+				pidCmd := []string{"pidof", emulator}
+				qemuPid, err := exec.ExecuteCommandOnPod(virtClient, readyPod, "compute", pidCmd)
 				// do not check for kvm-pit thread if qemu is not in use
 				if err != nil {
 					return
 				}
-				kvmpitmask, err := tests.GetKvmPitMask(readyPod, emulator, node)
+				kvmpitmask, err := tests.GetKvmPitMask(strings.TrimSpace(qemuPid), node)
 				Expect(err).ToNot(HaveOccurred())
 
 				vcpuzeromask, err := tests.GetVcpuMask(readyPod, emulator, "0")
 				Expect(err).ToNot(HaveOccurred())
 
 				Expect(kvmpitmask).To(Equal(vcpuzeromask))
-
 			},
-				Entry("[QUARANTINE] with explicit resources set", &virtv1.ResourceRequirements{
+				Entry("with explicit resources set", &virtv1.ResourceRequirements{
```

Comment:
Just a thought... What actually comes to my mind is that this previously quarantined test sets the resources for the vmi (cpu, mem) which are then setup for the virt-launcher pod. Since we run a bunch of commands (ps and such) inside the virt-launcher's compute container, might it be the case that they are just e.g. oom killed?